R/extractDocumentation.R

```diff
@@ -46,6 +46,12 @@ extractDocumentation <- function(path, start_type=NULL, comment="*'") {
     code <- "@(\\w*) ?(.*)$"
     pattern <- paste0("^(",escapeRegex(comment),") *",code)
     type <- sub(pattern,"\\2",x[1])
+    extraPage <- NULL
+    if (grepl("@code extrapage", x[1])) {
```

Comment:
please don't overwrite previous functionality but rather integrate it. Here, you overwrite the type argument and only catch a very specific case. the extrapage-Feature should work for any type, in particular it should also work for @equation